### PR TITLE
Adds Read to Container

### PR DESCRIPTION
**Pull request by cfmolinaforero**
Create Container Read Function

Output: Container, error

Description: This function performs the read action on the current instance of Container and returns a Container and an error (if there is one). Calls REST package to actually perform the read in Cosmos

### CODE REVIEW

File: container/container.go [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
-// Container defines the container client
+import "cosmos-go-sdk/rest"
+
+// Container defines the container
 type Container struct {
-	name   string
-	dbName string
-	key    string
+	Name    string
+	DbName  string
```
Comment (nicholasmfsmith) [inline container/container.go:13]: Do we need to store the name of the DB in the container?

Comment (cfmolinaforero) [reply container/container.go]: Hey man great catch, we should not need it as we are passing the uri upon creation. Made the corresponding changes.

File: container/container_test.go [hunk 2]
```diff
@@ -1,44 +1,59 @@
...
+			Request: mockRequest,
+		}
 	})
 
-	Context("Client", func() {
-		It("should successfully return a new instance of a Container Client", func() {
-			Expect(testClient).To(BeAssignableToTypeOf(Container{}))
+	Context("New", func() {
+		It("should successfully return a new instance of a Container", func() {
+			Expect(testContainer).To(BeAssignableToTypeOf(Container{}))
```
Comment (nicholasmfsmith) [inline container/container_test.go:33]: I think we forgot to add the New() method call here to validate what it is returning. Can we add it here?

```go
testContainer = New("testName", "testDatabaseURI", "testKey")
Expect(testContainer).To(BeAssignableToTypeOf(Container{}))
```

Comment (cfmolinaforero) [reply container/container_test.go:34]: Good catch, added!

**Pull request merged**